src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayRent = 40;
+  let sumForRent = 0;
+
+  if (days >= 7) {
+    sumForRent = oneDayRent * days - 50;
+
+    return sumForRent;
+  }
+
+  if (days >= 3) {
+    sumForRent = oneDayRent * days - 20;
```

Comment:
Similarly to the previous comment, replace the literal '20' with a constant that explains what this value represents.